LanguageFeatures/Parts-with-imports/import_inheritance_A01_t01.dart

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Import inheritance is a only suggestion: Aka. other files’
+/// imports cannot break your code (at least if you’re not depending on them). A
+/// part file is never restricted by the imports it inherits from its parent
+/// file. It can ignore and override all of them with imports of its own. That
+/// allows a file, like a macro generated file, to import all its own
+/// dependencies and be completely self-contained when it comes to imports.
+///
+/// @description Check that for the part file import, inherited from the parent,
+/// can be overridden by it's own import.
+/// @author sgrekhov22@gmail.com
+
```

Comment:
This might be helpful for future readers of this set of files:

```suggestion

/* A file relationship graph can be shown on github.com as follows:
\```mermaid
graph BT;
  Lib["import_inheritance_A01_t01.dart"]
  -->|import| PartsLib["parts_lib.dart\nlibVar, LibClass, libGetter, LibMixin"]
  Lib1["...lib1.dart\nlibVar, LibClass, foo, C"]
  Part1["...part1.dart"] -->|part of| Lib
  Part1 -->|import| Lib1
  Part2["...part2.dart"] -->|part of| Part1
  Lib2["...lib2.dart\nlibGetter, LibMixin, foo, C"]
  Part2 -->|import| Lib2
\```
*/
```

(I had to quote a couple of backquotes to avoid confusing the processing of a `suggestion`.)

Reply:
Thank you! Added to the `LanguageFeatures/Parts-with-imports/import_inheritance_A01_t01.dart` only. I don't think that it should be added to the each file. Please let me know if you disagree.